lua/methods/unit.lua

```diff
@@ -0,0 +1,14 @@
+--
+-- RPC unit actions
+-- https://wiki.hoggitworld.com/view/DCS_Class_Unit
+--
+
+GRPC.methods.getRadar = function(params)
+  local unit = Unit:getByName(params.name)

```

Comment:
Should we return a `GRPC.error("Unit not found")` if `unit` is `nil`?

Reply:
Done. While looking into this I saw that gRPC has built-in error codes for things like this ( https://grpc.github.io/grpc/cpp/namespacegrpc.html#aff1730578c90160528f6a8d67ef5c43b )

In this case we would want a `NOT_FOUND` code. Would it be possible to modify the `GRPC.Error` command to allow an optional way of setting the error code?